src/vmm_mad/remotes/lib/lxd/client.rb

```diff
@@ -119,19 +119,24 @@ def socket(lxd_path)
     def get_response(request, data)
         request.body = JSON.dump(data) unless data.nil?
 
-        request.exec(@socket, '1.1', request.path)
-
         response = nil
 
-        loop do
-            response = Net::HTTPResponse.read_new(@socket)
-            break unless response.is_a?(Net::HTTPContinue)
-        end
+        5.times do
```

Comment:
* Add a constant in the LXD in the LXDCLient class (API_RETRY) and change loop for `API_RETRY.times`
 * A break is missing in case of success to not do all the retries if it is not necessary
* if possible raise LXDError, better with a message ( `raise LXDError('Failed to read response from LXD'`)